customization/removing-template-vm-applications.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: doc
+title: Removing TemplateVM Packages
+permalink: /doc/removing-template-vm-packages/
```

Comment:
Let's make this `removing-templatevm-packages` for consistency (i.e., `templatevm` with no dashes).

Reply:
I think you forgot this part, @joshfleming.

Reply:
@andrewdavidwong You're right see the latest commit.